specification/search/data-plane/Azure.Search/preview/2020-06-30/searchservice.json

```diff
@@ -5656,17 +5657,22 @@
           {
             "value": "azureblob",
             "name": "AzureBlob",
-            "description": "Indicates a Azure Blob datasource."
+            "description": "Indicates an Azure Blob datasource."
           },
           {
             "value": "azuretable",
             "name": "AzureTable",
-            "description": "Indicates a Azure Table datasource."
+            "description": "Indicates an Azure Table datasource."
           },
           {
             "value": "mysql",
             "name": "MySql",
             "description": "Indicates a MySql datasource."
+          },
+          {
+            "value": "adlsgen2",
```

Comment:
This version of Search already GA'd and should not be changed.

Reply:
This would be a backwards compatible change of adding a new enum value, which I thought was permitted?

Reply:
Sorry, it is an extensible enum. GitHub was hiding it "perfectly", i.e. the JSON was valid with the elided section. I think this should be fine, then, since the value will roundtrip.